near/res/testnet_config.json

```diff
@@ -0,0 +1,208 @@
+{
+  "protocol_version": 4,
+  "genesis_time": "2019-06-04T06:13:25Z",
+  "chain_id": "testnet",
+  "num_block_producer_seats": 50,
+  "num_block_producer_seats_per_shard": [
+    7,
+    7,
+    6,
+    6,
+    6,
+    6,
+    6,
+    6
+  ],
+  "avg_hidden_validator_seats_per_shard": [
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0
+  ],
+  "dynamic_resharding": false,
+  "epoch_length": 600,
+  "gas_limit": 1000000000000000,
+  "min_gas_price": 5000,
+  "block_producer_kickout_threshold": 80,
+  "chunk_producer_kickout_threshold": 60,
+  "gas_price_adjustment_rate": 1,
+  "runtime_config": {
+    "storage_cost_byte_per_block": "1",
+    "poke_threshold": 60,
+    "transaction_costs": {
```

Comment:
We didn't have `transaction_costs` in the `testnet.json` before, seems like a some outdated config.

Reply:
We have this in near-node1. testnet.json has been outdated for a while because we had it bigger than github file size limit, and @bowenwang1996 maintain it separately

Reply:
@evgenykuzyakov we can update the config when we merge staging into master. Let's not do that in this PR.